admiral/debug_functions.sqf

```diff
@@ -287,6 +287,88 @@ adm_debug_fnc_updateLineMarker = {
     DEBUG("admiral.debug",FMT_3("Updated line marker '%1' position between '%2' and '%3'.",_markerName,_posFrom,_posTo));
 };
 
+adm_debug_fnc_createDebugCounterMarker = {
+    FUN_ARGS_3(_side,_type,_xPos);
+    
+    PVT_2(_marker,_markerType);
+    _markerType = if (_type != "total") then {
+        _type;
+    } else {
+        COUNTER_DEBUG_MARKER_TOTAL;
+    };
+    _marker = [format ["adm_counter_%1_%2", _side,_type], [_xPos, 50, 0], "ICON", _markerType, [_side] call adm_debug_fnc_getSideColor, COUNTER_DEBUG_MARKER_SIZE] call adm_common_fnc_createLocalMarker;
+    _marker setMarkerTextLocal "0";
+    
+    _marker;
+};
+
+adm_debug_fnc_createDebugFactionCounterMarkers = {
+    FUN_ARGS_2(_side,_xPos);
+
+    DECLARE(_debugMarkers) = [];
+    PVT_1(_marker);
+    {
+        _marker = [_side, _x, _xPos] call adm_debug_fnc_createDebugCounterMarker;
+        _debugMarkers pushBack _marker;
+        _xPos = _xPos + COUNTER_DEBUG_MARKER_X_INCREMENT;
+    } foreach GROUP_TYPE_DEBUG_MARKERS;
+    _marker = [_side, "total", _xPos] call adm_debug_fnc_createDebugCounterMarker;
+    _debugMarkers pushBack _marker;
+    _xPos = _xPos + COUNTER_DEBUG_MARKER_X_INCREMENT;
+    DEBUG("admiral.debug",FMT_2("Created counter Markers '%1' for side '%2'.",_debugMarkers,_side));
+};
+
+adm_debug_fnc_createAllDebugCounterMarkers = {
+    DECLARE(_xPos) = COUNTER_DEBUG_MARKER_X_POS;
+    {
+        [_x,_xPos] call adm_debug_fnc_createDebugFactionCounterMarkers;
+        _xPos = _xPos + (4 * COUNTER_DEBUG_MARKER_X_INCREMENT);
+    } foreach SIDE_ARRAY;
+};
+
+adm_debug_fnc_returnUnitCount = {
+    FUN_ARGS_1(_type);
+    
+    DECLARE(_aliveGroups) = switch (_type) do {
+        case (GROUP_TYPE_DEBUG_MARKERS select 0): {[[adm_cqc_groups, adm_patrol_infGroups,adm_camp_infGroups],_side] call adm_common_fnc_getAliveSideUnits;}; 
+        case (GROUP_TYPE_DEBUG_MARKERS select 1): {[[adm_patrol_techGroups, adm_camp_techGroups],_side] call adm_common_fnc_getAliveSideGroups;}; 
+        case (GROUP_TYPE_DEBUG_MARKERS select 2): {[[adm_patrol_armourGroups,adm_camp_armourGroups],_side] call adm_common_fnc_getAliveSideGroups;}; 
+        case "total": {[_side] call adm_common_fnc_getAllAliveSideUnits;};
+        default {[]};
+    };
+    
+    (str(count _aliveGroups));
```

Comment:
No need for parens here, since `count` is not a value it will `str` will be called with `count _aliveGroups`.